### DISCUSSION TITLE
how do you calculate lambda

### DISCUSSION THREAD

**Discussion by RRajdev** [moj-analytical-services/splink#1008] [Q&A] [answered]
hello, is the lambda a proportion of the initial count of records in the dataset or a function of the number of candidate pairs? i.e. if we have 100 records of which 10% are known duplicates, is the lambda: 

a) 10/100 which is 0.10
b) 
100(100-1)/2 = 4950  candidate pairs
10 duplicates 
10/4950 
0.002 

is this value therefore affected by blocking?

**Comment by RobinL** [comment, marked as answer]
There are 4950 possible choices. If it's true that 10 of these are dupes,  then 10/4950 is correct.

Note that if 10 of the records are duplicates and they're all the same person, then the '10' in the above fraction wouldn't be correct i.e. it matters how many distinct people there are

Probably the best two places to look are the following links.  Let me know if you need further clarification

https://moj-analytical-services.github.io/splink/demos/04_Estimating_model_parameters.html#estimation-of-probability_two_random_records_match

https://github.com/moj-analytical-services/splink/blob/7016703db36b8b936dd117991909706726b4983a/splink/files/settings_jsonschema.json#L17

**Comment by RRajdev** [comment]
thank you :)
